### PR TITLE
DASH-1596: Explicitly pass data through push notifications

### PR DESCRIPTION
**Pull request by anderson12ho**

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    then
+        {
+            twine upload dist/$WHEEL
+        } &> /dev/null
+        echo "Upload complete."
+    fi
+    {
+        rm -r build & rm -r dist
+    } &> /dev/null
+fi
```
Comment (owenbrownzc) [inline release.sh]: Generally, end files with newline.

Comment (owenbrownzc) [reply release.sh]: I'm guessing this is the resolution to the PyPi issue we were experiencing before?

**Review by owenbrownzc — APPROVED** [review]
Approved. My bash is weak, but I'm assuming you've run release.sh and it works as you intend.

Also, I'm assuming that the end receiver of the accepts the message with "data": {"url": ...} #.

File: release.sh (hunk 1 above)
Comment (anderson12ho) [reply release.sh]: This was what was used to generate the artifacts and upload to PyPi

Comment (anderson12ho) [reply release.sh]: An update to the previous version deployment to PyPi. The issue was as we suspected, a py3 artifact was not generated resulting in only downloading 3.8.
Once this is up hopefully my system will generate a 3.10 version

**Pull request merged**